recipes/doubtlab/meta.yaml

```diff
@@ -0,0 +1,154 @@
+{% set name = "doubtlab" %}
+{% set version = "0.1.1" %}
+
+
+##----------------------------------------------------------------------##
+## TODO for Maintainers:
+##
+## >> After each PyPI release of the package (doublab):
+## 
+##    1. [THIS GETS TAKEN CARE OF BY A BOT]: NO ACTION NEEDED
+##       Get the corresponding "version" and "sha256" of the latest 
+##       release and update the following (line number: 2 and 33).
+##       - version
+##       - source.sha256
+##
+##    2. [THIS IS A MANUAL PROCESS]
+##       Crosscheck and update the package dependency versions under:
+##       - requirements.host
+##       - requirements.run
+##
+## SEE THIS EXAMPLE: 
+## - how "pysteps" is maintaining "conda-forge/pysteps-feedstock":
+## https://pysteps.readthedocs.io/en/latest/developer_guide/update_conda_forge.html
+##
+##----------------------------------------------------------------------##
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/doubtlab-{{ version }}.tar.gz
+  sha256: e2065c916820ac6008883032594c14a0a660862cb012f3fac152a97f4c3c64b4
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - cleanlab >=1.0
+    - pandas >=1.3.3    
+    - scikit-learn >=1.0.0
+    # NOTE:
+    # tqdm is a dependency of cleanlab. But currently 
+    # (2021-11-15) conda-forge/cleanlab-feedstock  
+    # (CLFS) does not include it.
+    # A PR (#3) has been opened to address this issue.
+    # - https://github.com/conda-forge/cleanlab-feedstock/pull/3
+    # So, for now, tqdm is being included here in the 
+    # run-requirements. Also, tqdm is necessary for 
+    # passing pip check. However, once PR(#3) is merged
+    # with cleanlab-feedstock (CLFS), tqdm can be removed
+    # from the run-requirements.
+    #
+    # TODO: 
+    # - remove tqdm after cleanlab-feedstock (CLFS) PR#3 is merged.
```

Comment:
We probably only need line 62 from these comments.

Reply:
@ocefpaf Thank you. Suggestion adopted. Same as [**this comment**][#comment].

[#comment]: https://github.com/conda-forge/staged-recipes/pull/16940/files/d8d847786c94ca85203fd2678d1dd187425021a9#r757835861